systemd-units/plymouth-poweroff.sh.in

```diff
@@ -2,4 +2,5 @@
 
 if [ "$1" = "poweroff" ]; then
 	@PLYMOUTH_CLIENT_DIR@/plymouth quit --retain-splash
+	sleep 1
```

Comment:
Agree, but the 2.5.4 deadline doesn't give me enough time to fix this more carefully. Best thing here is probably to apply this workaround and try to find a better fix later.